tests/test_online_replacing.py

```diff
@@ -232,3 +237,37 @@ def testReadMultiple(self):
         response = requests.get("http://example.com")
         self.assertIn("This domain is for use", response.text)
         self.assertAlmostEqual(0.9974949866040544, sin_output, delta=0.0005)
+
+
+new_cassette = Cassette()
+
+
+@apply_decorator_to_all_methods(
+    replace_module_match(
+        what="math.sin", decorate=Simple.decorator_plain(), cassette=new_cassette
+    )
+)
+@apply_decorator_to_all_methods(record_requests(cassette=new_cassette))
```

Comment:
Can we use the shortcut?

```suggestion
@record_requests_for_all_methods(cassette=new_cassette)
```

Reply:
sure, at least it will test also this part